### PR TITLE
test(module:card): added missing tests

### PR DESCRIPTION
**Pull request by ParsaArvanehPA**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [✔] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [✔] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Application (the showcase website) / infrastructure changes
- [✔] Other... Please describe: missing unit tests for the Card Component

## What is the new behavior?
|               | Before  | After  |
|---------------|---------|--------|
| **Statements**|96.96%  | 100%   |
| **Branches**  | 100%   | 100%   |
| **Functions** | 85.71%  | 100%   |
| **Lines**     | 96.87%  | 100%   |

## Does this PR introduce a breaking change?
- [ ] Yes
- [✔] No

### CODE REVIEW

File: components/card/card.spec.ts [hunk 1]
```diff
@@ -166,3 +168,33 @@ export class NzTestCardRtlComponent {
   @ViewChild(Dir) dir!: Dir;
   direction: Direction = 'rtl';
 }
+
+describe('card', () => {
```
Comment (Laffery) [inline components/card/card.spec.ts:172]: test suite `card` duplicated, what about renaming it?

**Review by Laffery — CHANGES_REQUESTED** [review]
At line#34, you can make a change that provide icons with `provideNzIconsTesting()` in providrs instead of `NzIconModule.forRoot`

**Comment by ParsaArvanehPA** [comment]
@Laffery 
thank you for checking my PR, your rrequested changes are made.

File: components/card/card.spec.ts (hunk 1 above)
Comment (ParsaArvanehPA) [reply components/card/card.spec.ts]: yeah you'r right, done 👍

File: components/image/image-preview.component.ts
Comment (Laffery) [inline components/image/image-preview.component.ts:1]: I think you should split this file changes into a new PR

**Comment by ParsaArvanehPA** [comment]
@Laffery 
I have made the requested changes 👍

**Review by Laffery — APPROVED** [review]
LGTM

**Pull request merged**